lib/mkmf-rice.rb

```diff
@@ -66,7 +66,9 @@ def system_libffi_usable?
   libffi_ok
 end
 
-# Check for libffi to support C style callacks.
-if system_libffi_usable?
-  $CPPFLAGS += " -DHAVE_LIBFFI"
+def have_libffi
```

Comment:
How does this get called? Is there some convention in mkmf.rb that it calls methods that start with have_lib ? I thought you had to use `have_library` ?

Reply:
Projects that need this would call it in `extconf.rb`:

```ruby
require "mkmf-rice"

abort "libffi not found" unless have_libffi
```

(the name could be anything)